noetic/distribution.yaml

```diff
@@ -2662,6 +2662,23 @@ repositories:
       url: https://github.com/fkie/potree_rviz_plugin.git
       version: master
     status: maintained
+  flatland:
+    release:
+      packages:
+      - flatland
+      - flatland_msgs
+      - flatland_plugins
+      - flatland_server
+      - flatland_viz
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/avidbots/flatland-release.git
+      version: 1.3.1-2
```

Comment:
Release stanzas in rosdistro are usually added by Bloom. Is there a reason that this was added manually?

Reply:
My environment was poorly configured and was crashing when it went to push the diff after generating the release tags. I didn't want to generate a third release tag set so I just manually applied the diff. I'll have to figure out my environment one a dummy package sometime.

Reply:
I'll fix the per package license files and try again tomorrow.